classes/Language.php

```diff
@@ -984,6 +984,10 @@ public static function loadLanguages()
 				LEFT JOIN `' . _DB_PREFIX_ . 'lang_shop` ls ON (l.id_lang = ls.id_lang)';
 
         $result = Db::getInstance()->executeS($sql);
+        if (!is_array($result)) {
+            // executeS method can returns false
```

Comment:
```suggestion
            // executeS method can return false
```
That said, I think an exception might be more appropriate, because you should always have at least one language in database right?

Reply:
Unfortunately, during the install process, you don't have any languages in the database. :(